packages/angular/test/base/src/app/standalone/validation/input-validation/input-validation.component.html

```diff
@@ -0,0 +1,170 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Input - Validation Test</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="ion-padding">
+  <div class="validation-info">
+    <h2>Screen Reader Testing Instructions:</h2>
+    <ol>
+      <li>Enable your screen reader (VoiceOver, NVDA, JAWS, etc.)</li>
+      <li>Tab through the form fields</li>
+      <li>When you tab away from an empty required field, the error should be announced immediately</li>
+      <li>The error text should be announced BEFORE the next field is announced</li>
+      <li>Test in Chrome, Safari, and Firefox to verify consistent behavior</li>
+    </ol>
+  </div>
+
+  <form [formGroup]="form">
+    <div class="grid">
+      <div>
+        <h2>Required Email Field</h2>
+        <ion-input
+          #emailInput
+          id="email-input"
+          type="email"
+          label="Email"
+          labelPlacement="floating"
+          fill="outline"
+          placeholder="Enter your email"
+          [helperText]="fieldMetadata.email.helperText"
+          [errorText]="fieldMetadata.email.errorText"
+          formControlName="email"
+          required
+          [class.ion-touched]="isTouched('email')"
+          [class.ion-invalid]="isInvalid('email')"
+          [class.ion-valid]="isValid('email')"
```

Comment:
I suggest following how we implement the [Input value accessor](https://github.com/ionic-team/ionic-framework/blob/main/packages/angular/test/base/src/app/standalone/value-accessors/input) instead of manually adding these classes.